cognite/client/data_classes/hosted_extractors/sources.py

```diff
@@ -308,6 +308,54 @@ def _load_authentication(cls, resource: dict[str, Any]) -> Self:
         )
 
 
+@dataclass
```

Comment:
The class above `BasicMQTTAuthenticationWrite` is misleading as it should not be used for `MQTT` and only rest, as the `MQTT` has changed it seem.